starport/services/network/queries.go

```diff
@@ -117,3 +118,37 @@ func (n Network) GenesisValidators(ctx context.Context, launchID uint64) (genVal
 
 	return genVals, nil
 }
+
+// MainnetAccounts returns the list of campaign mainnet accounts for a launch from SPN
+func (n Network) MainnetAccounts(ctx context.Context, campaignID uint64) (genAccs []networktypes.MainnetAccount, err error) {
+	n.ev.Send(events.New(events.StatusOngoing, "Fetching campaign mainnet accounts"))
+	res, err := campaigntypes.NewQueryClient(n.cosmos.Context).MainnetAccountAll(ctx, &campaigntypes.QueryAllMainnetAccountRequest{
```

Comment:
```suggestion
	res, err := campaigntypes.
	NewQueryClient(n.cosmos.Context).
	MainnetAccountAll(ctx, &campaigntypes.QueryAllMainnetAccountRequest{
```

Can we do multi lines for this and similar funcs please? Code is getting out of the screen.

<img width="1481" alt="image" src="https://user-images.githubusercontent.com/6023117/155413182-1df3e8ff-de4d-48cf-b8d8-86a9113ddd38.png">